### ISSUE TITLE
Size is not working properly

### ISSUE THREAD

**Issue by atilahcS** [ERPedersen/nuxt-mdi#5]
Hi,
I've changed both size in `nuxt.config.ts` and by icon prop but the size is still not changed.
![image](https://github.com/ERPedersen/nuxt-mdi/assets/16699548/4c11c997-8c50-4079-89fb-fa6b522a9674)

**nuxt**: `3.11.1`
**@nuxtjs/tailwindcss**: `6.11.4`

Here's my config in `nuxt.config.ts`
```
  mdi: {
    cache: false,
    defaultSize: '4.5em',
  },
```

And here's in my template
```
<MdiIcon icon="mdiHandHeart" size="72px" />
```

**Comment by julius-spudvilas** [comment]
Confirming that it is a bug for me too.
Also size as  prop does not work ether...
Created PR https://github.com/ERPedersen/nuxt-mdi/pull/7 to fix the size from props issue.

Author should look into fixing size from CONFIG issue.


Logic for size should be :

Size from PROP || Size from CONFIG || Size from Default "1em"

**Comment by ERPedersen** [comment]
This has been fixed in v2.1.1
